metadata-ingestion/src/datahub/utilities/sqlglot_lineage.py

```diff
@@ -761,6 +761,54 @@ def sqlglot_lineage(
     default_db: Optional[str] = None,
     default_schema: Optional[str] = None,
 ) -> SqlParsingResult:
+    """Parse a SQL statement and generate lineage information.
+
+    This is a schema-aware lineage generator, meaning that it will use the
+    schema information for the tables involved to generate lineage information
+    for the columns involved. The schema_resolver is responsible for providing
+    the table schema information.
+
+    The parser supports most types of DML statements (SELECT, INSERT, UPDATE,
+    DELETE, MERGE) as well as CREATE TABLE AS SELECT (CTAS) statements. It
+    does not support DDL statements (CREATE TABLE, ALTER TABLE, etc.).
```

Comment:
Any plans for support here? I'm pretty sure I ran some DDL statements thru it the other day, is there a chance it worked or does it auto return an empty result?

Reply:
it won't crash on DDL statements, but it's not going to produce anything meaningful either

it might get some table lineage, and won't get column lineage for DDL statements

Reply:
Ah that's fine, I think this will mostly be useful for creating Operation aspects